.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on: [push]
+
+jobs:
+  test:
+    name: Test by running cookiecutter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: 'orbyter-cookiecutter'
+      - name: Install cookiecutter
+        run: pip install cookiecutter
+      - name: run ls
+        run: ls
+      - name: Cut a cookie (repo)
+        run: python -m cookiecutter --no-input orbyter-cookiecutter project_name=foundation repo_name=foundation
```

Comment:
I feel like it would be nice to have this command run in a docker container for development as future work (would be a nice follow on for this)

Reply:
That would involve adding more metadata and docker info. Plus, the typical use case I've seen for running the cookiecutter has been to run it on the bare machine, and doing it this way mimics that use case.

Reply:
I'm thinking about this use case (again, beyond the scope of this PR):

I make some changres to cookiecutter and want to see if it works. Currently, I:
1. push code to `origin`
2. Go to github.com and get the clone link
3. cd to some directory
4. `cookiecutter <clone_link> -b mws/mynewfeature` (`-b` may be wrong)
5. Edit changes in cookie cut repo to see desired state
6. Make changes in original repo
7. rinse and repeat

I wonder if steps 1-4 could be more streamlined. I think setting up a dev container and  running `python -m cookiecutter --no-input orbyter-cookiecutter ...` in it may help development. I may also just have a bad workflow :)

Reply:
Aha! I added #60 to track this.